.github/workflows/callable.test.yml

```
@@ -1,30 +1,45 @@
-        
+  workflow_call:
+    secrets:
+      SIGNING_SECRET:
+        required: true
+      SIGNING_PASSWORD:
+        required: true
+      SONAR_PROP_TOKEN:
+        required: true
+
 permissions:
     contents: read
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)